.buildkite/scripts/run_models.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -euo pipefail
```

Comment:
Is this line new? didn't see it in the initial update.

Reply:
Yep so this was re: tests yielding a false positive in integration testing. This is the fix that I will roll out to the rest of the migrations for last updates before migration branch release. Since we are releasing to main, we will make this update right now.